src/pds_doi_service/core/outputs/datacite/DOI_DataCite_template_20210520-jinja2.json

```diff
@@ -50,13 +50,32 @@
                         ],
                         {% endif %}
                         "nameIdentifiers": [
+                        {%- if author.rorid %}
+                            {
+                                "nameIdentifier": "{{ author.rorid }}",
+                                "nameIdentifierScheme": "ROR",
+                                "schemeUri": "https://ror.org"
+                            }{%- if author.orcid or author.name_identifiers %},{%- endif %}
+                        {%- endif %}
+                        {%- if author.orcid %}
+                            {
+                                "nameIdentifier": "{{ author.orcid }}",
+                                "nameIdentifierScheme": "ORCID",
+                                "schemeUri": "https://orcid.org"
+                            }
+                        {%- endif %}
                         {% for name_identifier in author.name_identifiers %}
                             {
                             {% for key, value in name_identifier.items() %}
                                 "{{key}}": "{{value}}"{% if not loop.last %},{% endif +%}
                             {% endfor %}
                             }
                         {% endfor %}
+                        ],
+                        "affiliations": [
```

Comment:
@rsjoyner There is a bug here. When I remove it, the tests pass. When I leave it, they fail.

```
$ python -m pytest src/pds_doi_service/core/outputs/test/datacite_test.py
...
FAILED src/pds_doi_service/core/outputs/test/datacite_test.py::DOIDataCiteValidatorTestCase::test_json_label_validation - pds_doi_service.core.entities.exceptions.InputFormatException: JSON record at index 0 do...
```